mycpp/pass_state.py

```diff
@@ -6,13 +6,42 @@
 import os
 from collections import defaultdict
 
+from mypy.types import Type
+from mypy.nodes import Expression
+
 from mycpp.util import join_name, log, SymbolPath
 
 from typing import Optional
 
 _ = log
 
 
+class ModuleMember(object):
+    def __init__(self, module_path: SymbolPath, member: str) -> None:
+        self.module_path = module_path
+        self.member = member
+
+
+class StaticObjectMember(object):
```

Comment:
Could use a comment for this ... I forgot that we also have AsdlType::CreateNull() -- I think that is the only instance of this!

Usually `::` is a C++ namespace generated for a Python module


----

Also I wonder if we should turn on type checking of mycpp in the CI?  Since we have more type annotations now